modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -898,11 +898,14 @@ definition {
 	}
 
 	@description = "LPS-94760 - Verify that when there is an alert message on the page, the update date should be displayed below it"
-	@ignore = "true"
 	@priority = 3
 	test SLAProcessingDateIsDisplayedUnderAlertMessage {
 		property test.name.skip.portal.instance = "WorkflowMetricsSLA#SLAProcessingDateIsDisplayedUnderAlertMessage";
 
+		WorkflowMetrics.setSLADefinitionsJobInterval(
+			definitionsJobInterval = 1,
+			slaJobInterval = 1);
+
```

Comment:
@victorhcunha can you give me context on why we need to add this step? Thank you!